### ISSUE TITLE
feature request - ufs needs a --device parameter

### ISSUE THREAD

**Issue by rhubarbdog** [ntoll/microfs#18]
I've got multiple microbits plugged into my laptop as I require multiple serial terminals.
I plugged my micropython microbit in first so it's `/dev/ttyACM0`. I then plugged in a MakeCode blocks microbit.
`ufs` defaults to the last plugged in microbit which is `/dev/ttyACM1` the MakeCode one. I need to be able to specify the device I choose to microfs into

**Comment by carlosperate** [comment]
Yes, now that microfs can send and receive files to different MicroPython-based devices, it's be good to add a `--port` flag to indicate the port manually, so that it can be used with any MicroPython board.
